### PR TITLE
mainet-005: Protocol versions update for Ecotone

### PR DESCRIPTION
**Pull request by sebastianst**
**Description**

Upgrades the recommended and required protocol versions to 6.0.0 (Ecotone) on mainnet.

This task is scheduled to run after [task-003](https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/eth/003-protocol-versions-delta).

- Fixes https://github.com/ethereum-optimism/protocol-quest/issues/156

### CODE REVIEW

File: tasks/eth/005-protocol-versions-ecotone/README.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# Mainnet Protocol Versions Update - 6.0.0 (Ecotone)
+
+Status: DRAFT, NOT READY TO SIGN
+
+## Objective
+
+This is the playbook for updating the required and recommended protocol versions of the `ProtocolVersions` contract on Ethereum mainnet to 6.0.0 (Ecotone).
```
Comment (mds1) [inline tasks/eth/005-protocol-versions-ecotone/README.md:7]: How can I verify that 6.0.0 is the correct version?

File: tasks/eth/005-protocol-versions-ecotone/README.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+   `m/44'/60'/1'/0/0` instead.
+
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+* For the `ProtocolVersions` Proxy at `0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935` both the
+  recommended and required storage slots are updated from the encoded form of `3.1.0` to `6.0.0`.
+  * key `0x4aaefe95bd84fd3f32700cf3b7566bc944b73138e41958b5785826df2aecace0`
+    * before: `0x0000000000000000000000000000000000000003000000010000000000000000` (before exectuion of task-003) or
```
Comment (mds1) [inline tasks/eth/005-protocol-versions-ecotone/README.md:76]: nit, this typo exists in a few places in this file
```suggestion
    * before: `0x0000000000000000000000000000000000000003000000010000000000000000` (before execution of task-003) or
```

File: tasks/eth/005-protocol-versions-ecotone/README.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    * before: `0x0000000000000000000000000000000000000005000000000000000000000000` (after exectuion of task-003)
+    * after : `0x0000000000000000000000000000000000000006000000000000000000000000`
+  * key `0xe314dfc40f0025322aacc0ba8ef420b62fb3b702cf01e0cdf3d829117ac2ff1a`
```
Comment (mds1) [inline tasks/eth/005-protocol-versions-ecotone/README.md:79]: Hm, I don't see this key in the simulation state changes, but I do see two `ConfigUpdated` events which look correct. Here's what I ran:
```
just install
forge clean
forge build
cd tasks/eth/005-protocol-versions-ecotone
export SIMULATE_WITHOUT_LEDGER=1
just simulate council
```

And here are the tenderly screenshots:

image

image

File: .circleci/config.yml [hunk 4]
```diff
@@ -168,6 +168,23 @@ jobs:
...
+      - run:
+          name: just simulate 005-protocol-versions-ecotone
+          command: |
+            just install
+            forge clean
+            forge build
+            cd tasks/eth/005-protocol-versions-ecotone
+            export SIMULATE_WITHOUT_LEDGER=1
+            just simulate council
+            just simulate foundation
```
Comment (mds1) [inline .circleci/config.yml:186]:
```suggestion
            just simulate
```

File: tasks/eth/005-protocol-versions-ecotone/README.md (hunk 3 above)
Comment (sebastianst) [reply tasks/eth/005-protocol-versions-ecotone/README.md:79]: I see the trace in the README screen shot if I just run `just simulate`.

Comment (mds1) [reply tasks/eth/005-protocol-versions-ecotone/README.md:79]: Tenderly confirmed this was a bug, and it's now resolved!

**Review by mds1 — APPROVED** [review]

**Pull request merged**